src/firefly/js/templates/lightcurve/LcPeriodogram.jsx

```diff
@@ -256,7 +257,7 @@ class PeriodogramOptionsBox extends Component {
                         </button>
                         <br/>
                         <div style={{marginTop: 10}}>
-                            {'Fields of "Period Min", "Period Max" and "Step Size" are allowed to be blank.'}
+                            {'Leave the fields blank to use default values.'}  <HelpIcon helpId={'periodogram.options'}/>
```

Comment:
Currently, the help button appears right under the help text. I think, it would be better if help button is with other buttons. In fact, FormPanel.jsx has a support for help button when you pass help_id property.

Also, there is too much white space under help text. It might be better if the help text was just above the lower form border.